### ISSUE TITLE
Bug: scheduled(observable, asapScheduler) not completing

### ISSUE THREAD

**Issue by benlesh** [ReactiveX/rxjs#7255]
After researching #7241, I've concluded that there's a bug where the following will not trigger the `complete` handler:

Tested in 7.8.0

```ts
scheduled(of('test'), asapScheduler).subscribe({
  next: (value) => {
    console.log('value received ' + value);
  },
  complete: () => {
    console.log('completed');
  },
});


// Expected
"value received test"
"completed"

// Actual
"value received test"
```

**Comment by benlesh** [comment]
This should be fixed with the publish of 7.8.1 (just published). Will confirm shortly.

**Comment by benlesh** [comment]
Confirmed fixed in latest (7.8.1)
